app/src/main/kotlin/cz/covid19cz/app/ui/login/LoginFragment.kt

```diff
@@ -131,9 +150,9 @@ class LoginFragment :
         timer.start()
     }
 
-    private fun showError(exception: Exception) {
-        login_info.text = exception.message
-        show(login_info)
+    private fun showError(text: Text?) {
```

Comment:
Proč používat Text? Není lepší si tam poslat jen resId?

Reply:
V momentálním stavu to určitě není možný, protože pro neznámý exceptiony tam pořád posílám jejich message. A i do budoucna je to podle mě lepší.